docusaurus/docs/Android/03-ui/06-guides/06-implementing-own-capabilities.mdx

```diff
@@ -0,0 +1,91 @@
+# Implementing Own Capabilities
+
+Each `Channel` has a list of capabilities that can be performed in it, such as muting a user, deleting messages and many
+others. Not all channels should have the same capabilities neither should all users, which is where own capabilities
+come into play.
+
+Inside of `Channel` you will find a property with a signature `ownCapabilities: Set<String>`. Which channel capabilities
+this set contains depends on the current user’s role (e.g. admin, guest, etc.), the channel type (e.g. messaging,
+livestream, etc.) and channel level settings. You can read more about channel
+capabilities [here](https://getstream.io/chat/docs/android/channel_capabilities/?language=kotlin) and more about channel
+level settings [here](https://getstream.io/chat/docs/android/channel-level_settings/).
+
+## Integrating channel capabilities into the UI
+
+If you are using our own UI in combination with our `ViewModel`s, then own capabilities work out of the box without any
```

Comment:
`your own` I imagine? (typo)

Reply:
The intent was to signal that our complete solution (UI + VM) already has own capabilities implemented without any additional work.

Reply:
I think we can then omit `own` as it's implied